### DISCUSSION TITLE
It needs to be more functionality such as scientific and programmers calculator

### DISCUSSION THREAD

**Discussion by nazares** [elementary/calculator#265] [General]
It is not so cool to be the default calculator for the whole OS  with that poor functionality. I would say nothing if it would be another one kid's handmade calculator. But you  position this as the default tool for ElementaryOS.

**Comment by jeremypw** [comment]
That may be so, but we need a list of specific features, preferably prioritised, so that issues can be raised against each.  Calculator provides most of the *basic* mathematical functions like arithmetic, trigonometric and logarithmic as well as memory and parentheses. Some are hidden behind the extended functionality button.
